### DISCUSSION TITLE
Disable AI Conversations in Sub Test Mode

### DISCUSSION THREAD

**Discussion by Jlobblet** [FakeFishGames/Barotrauma#5155] [Ideas and suggestions]
When in test mode (accessed via the sub editor) AIs should not have conversations.

**Comment by 3e849f2e5c** [comment]
Easy enough to implement, consider it done

**Comment by Krzeszny** [comment]
I think you've misspelled the title. It should be "Let us disable AI conversations in singleplayer mode". I'll probably have to post my own discussion :/
